src/Widgets/ThumbnailGalleryWidget/ThumbnailGalleryWidgetComponent.js

```diff
@@ -124,19 +179,36 @@ function allTags(images) {
   return uniqueTags.sort();
 }
 
-function lightboxOptions(galleryImageWidget) {
-  const image = galleryImageWidget.get("image");
-  const binary = image.get("blob");
-  const srcUrl = binary.optimizeFor({ width: fullScreenWidthPixels() }).url();
-  const alt = image.get("alternativeText");
-
-  return {
-    photo: srcUrl,
-    thumbnail: srcUrl,
-    caption: [
-      galleryImageWidget.get("title"),
-      galleryImageWidget.get("subtitle"),
-    ].join(" - "),
-    alt,
-  };
+function SliderNextArrow(props) {
+  return (
+    <button className="slider-next-arrow" onClick={props.onClick}>
+      <svg
+        viewBox="0 0 18 18"
+        aria-hidden="true"
+        style={{ height: "2.8em", width: "2.8em", fill: "#fff" }}
+      >
+        <path
+          d="M4.29 1.71A1 1 0 1 1 5.71.3l8 8a1 1 0 0 1 0 1.41l-8 8a1 1 0 1 1-1.42-1.41l7.29-7.29z"
+          fillRule="evenodd"
+        />
+      </svg>
```

Comment:
* [ ] Q: Since we include FontAwesome in the app anyways, is there a particular reason why we don't use the icons here?

Reply:
No particular reason. Good idea to make use of font awesome.